source/faq.txt

```diff
@@ -30,6 +30,11 @@ If you write to a synced namespace before issuing a :ref:`commit
 undefined. To ensure that you don't write to any synced namespaces, enable 
 :ref:`write blocking <c2c-write-blocking>`.
 
+.. note::
+   
+   Index builds on the destination cluster are treated as writes 
+   while ``mongosync`` is syncing.
```

Comment:
CC @ajayvijayakumar123 this was added in response to the discussion around rolling index builds. Please LMK if there's anything else that needs to be added - thanks!

Reply:
this sounds good!